.github/workflows/qns-comment.yml

```diff
@@ -17,40 +17,8 @@ jobs:
     permissions:
       pull-requests: write
     runs-on: ubuntu-latest
-    if: >
-      github.event.workflow_run.event == 'pull_request' &&
-      github.event.workflow_run.conclusion == 'failure'
+    if: github.event.workflow_run.event == 'pull_request'
```

Comment:
In the case of qns, the comment should still only be printed on `failure`, right?

Reply:
Yes, this needs to be restored - I was just trying to make it generate any output.